src/main/java/gel/Ui.java

```diff
@@ -0,0 +1,63 @@
+package gel;
+
+import java.util.List;
+
+import gel.task.Task;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+
+    public void showLoadingError() {
+        System.out.println("    Lol... Did not manage to load storage data..."
+                + " Creating new TaskList now...");
+    }
+
+    public void showWelcomeMessage() {
+        System.out.println("    Hello! I'm Gel\n    What do you want from me?\n");
+    }
+
+    public void farewellMessage() {
```

Comment:
Should this method name be in the form of a verb instead? E.g. `printFareWellMessage()`. See [Java Coding Standard -> Naming](https://se-education.org/guides/conventions/java/index.html#naming). I notice the same issue a few times in this file as well.